op-chain-ops/genesis/config.go

```diff
@@ -730,6 +730,7 @@ type L2InitializationConfig struct {
 	UpgradeScheduleDeployConfig
 	L2CoreDeployConfig
 	AltDADeployConfig
+	DAFootprintGasScalar uint16 `json:"daFootprintGasScalar"`
 }
```

Comment:
It looks fine to me to just add the DA footprint gas scalar directly as a field, instead of to an existing or new config struct, since all possible values are valid.

I'm wondering though if there's an assumption anywhere that all fields of the `L2InitializationConfig` are config structs instead of direct config values.